autonomy/chain/constants.py

```diff
@@ -29,81 +31,93 @@
-SERVICE_MANAGER_ADDRESS_GOERLI = "0x1d333b46dB6e8FFd271b6C2D2B254868BD9A2dbd"
-SERVICE_REGISTRY_ADDRESS_GOERLI = "0x1cEe30D08943EB58EFF84DD1AB44a6ee6FEff63a"
-GNOSIS_SAFE_PROXY_FACTORY_ADDRESS_GOERLI = "0x65dD51b02049ad1B6FF7fa9Ea3322E1D2CAb1176"
-GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS_GOERLI = (
-    "0x92499E80f50f06C4078794C179986907e7822Ea1"
-)
-SERVICE_REGISTRY_TOKEN_UTILITY_ADDRESS_GOERLI = (
-    "0x6d9b08701Af43D68D991c074A27E4d90Af7f2276"  # nosec
-)
-MULTISEND_ADDRESS_GOERLI = "0x40A2aCCbd92BCA938b02010E17A5b8929b49130D"
-
-# contract addressed for a custom chain
-COMPONENT_REGISTRY_ADDRESS_CUSTOM = os.environ.get("CUSTOM_COMPONENT_REGISTRY_ADDRESS")
-AGENT_REGISTRY_ADDRESS_CUSTOM = os.environ.get("CUSTOM_AGENT_REGISTRY_ADDRESS")
-REGISTRIES_MANAGER_ADDRESS_CUSTOM = os.environ.get("CUSTOM_REGISTRIES_MANAGER_ADDRESS")
-SERVICE_MANAGER_ADDRESS_CUSTOM = os.environ.get("CUSTOM_SERVICE_MANAGER_ADDRESS")
-SERVICE_REGISTRY_ADDRESS_CUSTOM = os.environ.get("CUSTOM_SERVICE_REGISTRY_ADDRESS")
-GNOSIS_SAFE_PROXY_FACTORY_ADDRESS_CUSTOM = os.environ.get(
-    "CUSTOM_GNOSIS_SAFE_PROXY_FACTORY_ADDRESS"
+
+@dataclass
+class ContractAddresses:  # pylint: disable=too-many-instance-attributes
+    """Contract addresses container for a chain."""
+
+    component_registry: str
+    agent_registry: str
+    registries_manager: str
+    service_registry: str
+    service_registry_token_utility: str
+    service_manager: str
+    gnosis_safe_proxy_factory: str
+    gnosis_safe_same_address_multisig: str
+    multisend: str
+
+    def get(self, name: str) -> str:
+        """Returns the contract address."""
+        return cast(str, getattr(self, name))
+
+
+HardhatAddresses = ContractAddresses(  # nosec
+    component_registry="0x5FbDB2315678afecb367f032d93F642f64180aa3",
+    agent_registry="0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
+    registries_manager="0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0",
+    service_registry="0x998abeb3E57409262aE5b751f60747921B33613E",
+    service_manager="0x4c5859f0F772848b2D91F1D83E2Fe57935348029",
+    gnosis_safe_proxy_factory="0x0E801D84Fa97b50751Dbf25036d067dCf18858bF",
+    gnosis_safe_same_address_multisig="0x8f86403A4DE0BB5791fa46B8e795C547942fE4Cf",
+    service_registry_token_utility="0x36C02dA8a0983159322a80FFE9F24b1acfF8B570",
+    multisend="0x9d4454B023096f34B160D6B654540c56A1F81688",
 )
-GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS_CUSTOM = os.environ.get(
-    "CUSTOM_GNOSIS_SAFE_SAME_ADDRESS_MULTISIG_ADDRESS"
+
+EthereumAddresses = ContractAddresses(  # nosec
+    component_registry="0x15bd56669F57192a97dF41A2aa8f4403e9491776",
+    agent_registry="0x2F1f7D38e4772884b88f3eCd8B6b9faCdC319112",
+    registries_manager="0x9eC9156dEF5C613B2a7D4c46C383F9B58DfcD6fE",
+    service_registry="0x48b6af7B12C71f09e2fC8aF4855De4Ff54e775cA",
+    service_registry_token_utility="0x3Fb926116D454b95c669B6Bf2E7c3bad8d19affA",
+    service_manager="0x2EA682121f815FBcF86EA3F3CaFdd5d67F2dB143",
```

Comment:
it seems here that you call `service_manager` also on L1. Should we call this with correct name: service_manager_token?

Reply:
I'm storing both `service_manager` and `service_manager_token` as `service_manager` to avoid complications in the code